runtime/common/src/precompile/schedule_call.rs

```diff
@@ -213,12 +185,18 @@ impl<
 					task_id,
 				);
 
-				Scheduler::schedule_named(
+				<pallet_scheduler::Pallet<Runtime> as ScheduleNamed<
+					BlockNumber,
+					<Runtime as pallet_scheduler::Config>::Call,
+					<Runtime as pallet_scheduler::Config>::PalletsOrigin,
+				>>::schedule_named(
 					task_id.clone(),
 					DispatchTime::After(min_delay),
 					None,
 					0,
-					Origin::root().caller().clone(),
+					<<<Runtime as pallet_scheduler::Config>::Call as Dispatchable>::Origin>::root()
```

Comment:
do we need use `ScheduleOrigin` inside `pallet_scheduler::Config`, or just use `root()` here?

Reply:
this is fine. later we may want to create an EVM Scheduler origin if we want to avoid use root here.
but user have no control on the call so it is safe.